vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java

```diff
@@ -386,6 +386,9 @@ else if (OWNING_BOARD.equals(key)) {
           owningBoardName = null;
         }
       }
+      else if (value.equals("")) {
+        owningBoardName = null; // Because "" is what null sometimes turns into during Vassal configurers
+      }
```

Comment:
I don't believe this is needed once you add the postPasteFixes() call in. Just adding the postPasteFixes() call fixed the issue for me.

Reply:
Still needed -- see comment above.